client/components/repository/common/Discussion.vue

```diff
@@ -0,0 +1,74 @@
+<template>
```

Comment:
Consider having separate wrappers for editor and elements (since their style, behaviour and mappings might be different)